bugbug/model.py

```diff
@@ -133,7 +137,40 @@ def get_important_features(self, cutoff, shap_values):
         # Sort the array (in decreasing order of importance values)
         top_features = top_features[top_features[:, 0].argsort()][::-1]
 
-        return top_features
+        if isinstance(shap_values, list):
+            important_features = {}
+            important_features["classes"] = {}
+            important_features["average"] = list(
+                int(index) for importance, index, is_pos in top_features
+            )
+            for num, item in enumerate(shap_values):
+                # top features in shap values for that class
+                top_item_features = self.get_important_features(
+                    cutoff, item, item, class_names
+                )
+
+                # top features in average shap values for that class
+                top_avg = []
+                abs_sums = np.abs(item).sum(0)
+                rel_sums = abs_sums / abs_sums.sum()
+                is_pos = ["+" if shap_sum >= 0 else "-" for shap_sum in item.sum(0)]
+                for index in important_features["average"]:
+                    top_avg.append(is_pos[index] + str(rel_sums[index]))
+
+                if class_names:
+                    important_features["classes"][
+                        class_names[len(class_names) - num - 1]
```

Comment:
Are you sure they are in reverse order?

Reply:
Actually not in reverse order but currently they are already reversed in model.py so i reversed them again.